etc/testing/arangosh.conf

```diff
@@ -1,6 +1,18 @@
+[console]
+pretty-print = true
+
+[server]
+authentication = false
+# username = root
+# password =
+
 [log]
 file = -
 
 [javascript]
+startup-directory = ./js
 allow-external-process-control = true
 allow-port-testing = true
+
+[javascript:enterprise]
```

Comment:
This syntax with a colon is new to me. Does it also work on the command line? Are there any other options like this?

Reply:
Only `:enterprise` and `:community`:
https://github.com/arangodb/arangodb/blob/devel/lib/ProgramOptions/IniFileParser.cpp#L52-L57